src/components/features/PromotedBox/PromotedBox.module.scss

```diff
@@ -0,0 +1,256 @@
+        font-size: 0;
+        @extend %transition-animation;
+
+        &.active,
+        &:hover {
+          background-color: $primary;
+        }
+      }
+    }
+  }
+}
+
+.photoBox {
+  position: relative;
+  height: 300px;
+  width: 100% ;
+
+  .images {
+    position: absolute;
+    object-fit: cover;
+    padding: 0;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+    z-index: 0;
+  }
+
+  .buttons {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    z-index: 1;
+    transform: translate(-50%, -50%);
+    visibility: hidden;
+    opacity: 0;
+    transition: all 0.5s ease-in;
+  }
+
+  .timerWrapper {
+    position: absolute;
+    display: inline-flex;
+    justify-content: space-evenly;
+    z-index: 15;
+    bottom: 10px;
+    width: 100%;
+    padding: 0 15px 0 15px;
+    visibility: hidden;
+    opacity: 0;
+    transition: all 0.5s ease-in;
+  }
+
+  .timerText {
+    background-color: $header-bg;
+    border-radius: 50px;
+    width: 70px;
+    height: 70px;
+    color: $background-white;
+    text-align: center;
+    justify-content: center;
+    padding: 5px;
+    font-weight: 600;
+    .textBig {
+      font-size: 25px;
+    }
+
+    .textSmall {
+      font-size: 12px;
+    }
+  }
+}
+
+.photoBox:hover {
+  .buttons{
+    visibility: visible;
+    opacity: 1;
+
+  }
+  .timerWrapper {
+    visibility: visible;
+    opacity: 1;
+  }
+}
+
+.content {
+  text-align: center;
+  padding: 20px;
+
+  h5 {
+    color: $color-black;
+    font-size: 16px;
+    line-height: 24px;
+    letter-spacing: 1px;
+  }
+}
+
+.line {
+  margin: 0 10px;
+  border-bottom: 1px solid $footer-claim-bg;
+  position: relative;
+
+  &::before {
+    content: "";
+    position: absolute;
+    bottom: -1px;
+    left: 50%;
+    width: 30px;
+    border-bottom: 4px solid $primary;
+    transform: translateX(-50%);
+  }
+}
+
+.actions {
+  padding: 15px 10px 10px 10px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.priceold {
+  color: $text-price-old;
+  font-size: 16px;
+  text-decoration: line-through;
+  background: none;
+  padding-left: $base-padding * 2;
+}
+
+.productBox {
+  background-color: $background-white;
+  border: 1px solid $product-box-border-color;
+  width: 350px;
+}
+
+.topBar {
+  display: flex;
+  background-color: $header-bg;
+  height: 50px;
+  align-items: center;
+}
+
+.title {
+  color: $background-white;
+  font-size: 20px;
+  font-weight: 600;
+  vertical-align: center;
+}
+
+.promoWrapper {
+  position: relative;
+  border: 1px solid $product-box-border-color;
+
+  height: 100%;
+  .imagePromo {
+    height: 100%;
+    width: 100%;
+    object-fit: cover;
+    z-index: 1;
+  }
+}
+
+.banner {
+  display: block;
+  content: "";
+  color: $background-white;
+  position: absolute;
+  top: 45%; left: 50%;
+  transform: translate(-50%,-50%);
+  z-index: 2;
+  height: 250px;
+  width: 100%;
+  background-color: $header-topbar-bg;
+  opacity: 0.9;
```

Comment:
ja bym zmniejszył opacity na 0.6 / 0.7 i zrobił to tak, żeby biały tekst, który jest w tym elemencie miał opacity: 1; Będzie lepiej widać zdjęcie, które jest pod spodem i będzie to wtedy wyglądało jak w designie